handlers/eth1Account.go

```diff
@@ -147,29 +147,11 @@ func Eth1Address(w http.ResponseWriter, r *http.Request) {
 	})
 	g.Go(func() error {
 		var err error
-		addressWithdrawals, nextPageToken, err := db.GetAddressWithdrawals(addressBytes, 25, "")
+		withdrawals, err = db.GetAddressWithdrawals(addressBytes, 25, "", currency)
```

Comment:
> we want a `db.BigtableClient.GetAddressWithdrawalsTableData(addressBytes, "", currency)` here.

I am not sure that I can move the function to `BigtableClient`. The data does not seem to come from BigTable. It queries a database like so: `ReaderDb.Select(&withdrawals, ...` and `ReaderDb` is of type `*sqlx.DB`.
Maybe I misunderstand you or the code :)

I renamed the function according to your suggestion.

I also changed all occurrences of `25` into a new constant `DefaultSizeOfRowsQueries`. It was not in the BIDS, I hope this will not cause difficulties.

Regarding parameter `currency string`, I agree that it feels abnormal. However, the currency is requested in the URL, I think that we cannot use a constant. Functions in _db/bigtable_eth1.go_ use constant `utils.Config.Frontend.ElCurrency`.

Reply:
**bigtable_eth1.go:** You are right, I thought the withdrawals function gets some data from bigtable but it does not, therefore it shouldn't be moved.

**DefaultSizeOfRowsQueries:** Thanks, I just don't quite like the name of it since it seems too broadly defined.
It is only used for the amount of rows of datatables that have an infinite scroll so how about something like `DefaultInfScrollRows`?

**currency:** I tested the behaviour a bit and you can see that for e.g. https://beaconcha.in/address/0x969499ecc1abb0334ecfdf18d7135d6681189882 when you change the currency in the top right all the tabs stay at ETH except the withdrawals.
We don't want a behaviour change for that for now so the `currency` paramter is fine.